ci-operator/config/openshift/operator-framework-rukpak/openshift-operator-framework-rukpak-main.yaml

```diff
@@ -42,6 +42,22 @@ tests:
   commands: make verify
   container:
     from: src
+- as: verify-deps
+  commands: |
+    # Verify main repo deps
+    make -f openshift/Makefile verify-deps
```

Comment:
I'm not seeing this make target in the repo, is it something you're in the process of adding?

Reply:
It's part of the build-machinery: https://github.com/openshift/operator-framework-rukpak/blob/main/openshift/Makefile#L3

Reply:
By the transitive process of [lots-of-includes]...
https://github.com/openshift/operator-framework-rukpak/blob/main/openshift/vendor/github.com/openshift/build-machinery-go/make/targets/openshift/deps-gomod.mk